.gitpod.yml

```diff
@@ -65,8 +65,11 @@ tasks:
       leeway exec --package components/ide/jetbrains/backend-plugin:plugin-latest --package components/ide/jetbrains/gateway-plugin:publish-latest --parallel -- ./gradlew buildPlugin
   - name: TypeScript
     init: yarn --network-timeout 100000 && yarn build
+  - name: Install pre-commit hooks
```

Comment:
Not sure why this was added to `Go` in the beginning. 🤷🏻‍♂️

Reply:
:shrug: